### ISSUE TITLE
`Run in Neo4j Browser` gives out no notice when Neo4j Desktop is not installed

### ISSUE THREAD

**Issue by stefano-ottolenghi** [neo4j-documentation/docs-ui#141]
The run buttons only have an effect if Neo4j Desktop is installed, as they go through the `neo4j-desktop://` protocol. Not everybody has it, and as it currently stands it's impossible to know that that's a prerequisite. We should alert users when clicking, if the protocol is unavailable.

**Comment by adam-cowley** [comment]
I don't think you can detect whether the software is installed.  We could instead update these links to point to `browser.neo4j.io?cmd=edit&arg=url%20encoded%20cypher%20statement` instead

**Comment by stefano-ottolenghi** [comment]
It looks like you are right: you can't inquire whether a protocol has a handler registered in the browser, I guess for privacy reasons. I just assumed you could.

> We could instead update these links to point to [browser.neo4j.io](http://browser.neo4j.io/)

Unless we can also automatically spin up an instance for them, so that the link actually _runs_ the query, I don't think it would help much. You land there and it's not at all easy to understand that you first need to get a running instance, connect to it (with another query), and then come back to the button.

As an alternative, I feel like the best we could do is put a clarifying note on hover?
